crates/red_knot_python_semantic/resources/mdtest/conditional/match.md

```diff
@@ -48,7 +48,7 @@ def _(target: int):
 
 ```py
 class NotBoolable:
-    __bool__ = 3
+    __bool__: int = 3
```

Comment:
Without these changes, we infer the type of `__bool__` to be `int | Unknown`. Only one of the union branches is non-callable, which changes the content of the error message below. I decided to add the type annotations instead of updating the expected error messages, since this seems to more accurately reflect the intent of these `NotBoolable` types.